dolphinscheduler-ui/src/service/modules/process-definition/index.ts

```diff
@@ -255,3 +256,10 @@ export function viewProcessDefinitionVariables(code: number, processCode: number
   })
 }
 
+export function batchOnlineByCodeStates(data: CodeStateListReq, code: number): any {
+    return axios({
+        url: `/projects/${code}/process-definition/batch-release`,
+        method: 'post',
+        data
+    })
+}
```

Comment:
We suggest the last line to be blank, so please add a new line here.

Reply:
Okay, I'll make the necessary modifications